app/models/concerns/notifications/enum_status_values.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength, Metrics/MethodLength
+module Notifications
+  module EnumStatusValues
+    extend ActiveSupport::Concern
+
+    ACTIVEDUTY = 'activeduty'
+    CANCELED_DECLINED = 'canceled_declined'
+    CLOSED = 'closed'
+    DECEASED = 'deceased'
+    ENROLLED = 'enrolled'
+    INELIG_CHAMPVA = 'inelig_champva'
+    INELIG_CHARACTER_OF_DISCHARGE = 'inelig_character_of_discharge'
+    INELIG_CITIZENS = 'inelig_citizens'
+    INELIG_FILIPINOSCOUTS = 'inelig_filipinoscouts'
+    INELIG_FUGITIVEFELON = 'inelig_fugitivefelon'
+    INELIG_GUARD_RESERVE = 'inelig_guard_reserve'
+    INELIG_MEDICARE = 'inelig_medicare'
+    INELIG_NOT_ENOUGH_TIME = 'inelig_not_enough_time'
+    INELIG_NOT_VERIFIED = 'inelig_not_verified'
+    INELIG_OTHER = 'inelig_other'
+    INELIG_OVER65 = 'inelig_over65'
+    INELIG_REFUSEDCOPAY = 'inelig_refusedcopay'
+    INELIG_TRAINING_ONLY = 'inelig_training_only'
+    LOGIN_REQUIRED = 'login_required'
+    NONE = 'none_of_the_above'
+    PENDING_MT = 'pending_mt'
+    PENDING_OTHER = 'pending_other'
+    PENDING_PURPLEHEART = 'pending_purpleheart'
+    PENDING_UNVERIFIED = 'pending_unverified'
+    REJECTED_INC_WRONGENTRY = 'rejected_inc_wrongentry'
+    REJECTED_RIGHTENTRY = 'rejected_rightentry'
+    REJECTED_SC_WRONGENTRY = 'rejected_sc_wrongentry'
+
+    class_methods do
+      # Creates the ActiveRecord::Enum mappings between the attribute values and
+      # their associated database integers.
+      #
+      # To add a new value, add it to the **end** of the hash, incrementing the integer.
+      #
+      # Do **NOT** remap any existing attributes or integers.
+      #
+      # @return [Hash]
+      # @see https://api.rubyonrails.org/v5.2/classes/ActiveRecord/Enum.html
+      #
+      def statuses_mapped_to_database_integers
```

Comment:
why is this a method instead of a constant?

Reply:
Yeah, here was my thinking for making this a method vs. a constant:

- A method naturally lends itself to having structured Yard documentation, and in this case, we definitely want to have extensive documentation on how to maintain this structure, as it creates the database mappings between the values and their respective db integers
- it feels like more of a method to me, since we're taking action on the hash by calling a method on it, prior to returning it 

I see your point, as this structure will remain relatively constant over time. Based on the rationale above, I lean towards defining it in a method.

That work?

Reply:
constants can be documented in yard
the symbolize keys method can be called when defining the constant `FOO = {}.symbolize_keys`